tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -168,7 +163,7 @@ def test_multiple_efs(
     for i in range(num_existing_efs):
         existing_efs_mount_dirs.append(f"/existing_efs_mount_dir_{i}")
 
-    new_efs_mount_dirs = ["/shared"]  # OSU benchmark relies on /shared directory
+    new_efs_mount_dirs = ["/shared"]
```

Comment:
According to the removed comment it seems that the /hsared folder was required only by OSU benchmarks. Is it still required if we remove them?

Reply:
Yes, I originally tried to remove the "/shared", but the cluster configs use the shared mount dir ([test_efs here](https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/tests/storage/test_efs/test_multiple_efs/pcluster.config.yaml#L46-L53) and [test_fsx_lustre here](https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/tests/storage/test_fsx_lustre/test_multiple_fsx/pcluster.config.yaml#L38-L46)), and I wasn't sure if I should edit those cluster configs to delete them if the mount dir was used for something else, so I added the "/shared" back in. Should I try deleting them?

Reply:
No, in this case better to keep it.